modules/common/src/middleware/elapsed-time.middleware.ts

```diff
@@ -23,19 +23,19 @@ export class ElapsedTimeMiddleware implements NestMiddleware {
         }
 
         const startHrTime: [number, number] = process.hrtime();
-        const requestIdentifier: string = `${req.method} ${req.originalUrl}`;
         res.on('finish', (): void => {
             const elapsedHrTime: [number, number] = process.hrtime(startHrTime);
             const elapsedTimeInMs: number = elapsedHrTime[0] * 1e3 + elapsedHrTime[1] / 1e6;
-            this.logger.logObject({
+            this.logger.log({
                 statusCode: `${res.statusCode}`,
-                request: requestIdentifier,
                 elapsedTime: `${elapsedTimeInMs}`,
-                message: `Route finished: ${requestIdentifier} ${res.statusCode}, execution time ${elapsedTimeInMs}ms`,
+                message: 'Route finished',
             });
         });
 
-        this.logger.log(`Route started: ${requestIdentifier}`);
+        this.logger.log({
+            message: 'Route started',
+        });
```

Comment:
Route started doesn't have `httpRequest` object...
![Screenshot 2021-09-06 at 16 15 24](https://user-images.githubusercontent.com/156132/132230438-b7ead545-a02a-423b-b740-34079fe251af.png)

![Screenshot 2021-09-06 at 16 14 37](https://user-images.githubusercontent.com/156132/132230330-2fa52d65-e40a-461c-ae98-2782955e90c9.png)

Reply:
and why to submit it as object if you can use just text?

Reply:
We can just use text yes
For the httpResponse easiest solution is to have request middleware first so it is saved in httpContext first.

Reply:
in such case, it would not have the request ID or? did you test it? We for sure need both, request data and request ID as part of it...

Reply:
Right now it is deployed with the new middleware order and route started has both httpResponse and requestId